src/views/Home/Home.test.jsx

```diff
@@ -0,0 +1,35 @@
+import Home from './';
+import { render, screen } from '@testing-library/react';
+import { AppWrapper } from '../../utils/testing';
+import banners from '../../mocks/en-us/featured-banners.json';
+import categories from '../../mocks/en-us/product-categories.json';
+import products from '../../mocks/en-us/featured-products.json';
+
+beforeEach(() => {
+  render(
+    <AppWrapper>
+      <Home />
+    </AppWrapper>
+  );
+});
+
+describe('Featured Banners', () => {
+  it('renders data from API', async () => {
+    const bannerTitle = await screen.findByRole('heading', { name: banners.results[0].data.title });
+    expect(bannerTitle).toBeInTheDocument();
+  });
+});
+
+describe('Carousel', () => {
+  it('renders data from API', async () => {
+    const categoryName = await screen.findByText(new RegExp(categories.results[0].data.name));
+    expect(categoryName).toBeInTheDocument();
```

Comment:
We are not checking that all the categories are displayed here. Querying for each category name and image are displayed correctly would be something like:
```suggestion
   for(const category of categories.results){
      expect(await screen.findByRole("img",{name:category.data.main_image.alt})).toBeInTheDocument()
      expect(await screen.findByText(category.data.name)).toBeInTheDocument()
    }
```

Reply:
Sure, thanks